spec/fixtures/static_pages/mp.php?mpn=Christine_Milne&mpc=Tasmania&house=senate&display=everyvote.html

```diff
@@ -106,14 +106,14 @@ <h2 class="social-share-heading small">
 
 <h2 id="divisions">All Votes</h2>
 <p>All votes Senator Christine Milne could have attended.</p>
-<ol class="display-house-all divisions-list list-unstyled">
+<ol class="display-house-senate divisions-list list-unstyled">
```

Comment:
All these static html regression tests are to make sure that we're not accidentally changing the behaviour. The reason for this is that this app was converted over from the very old PHP code base. Some of the logic is very complex and there's lots of different stuff going on, and there's not a lot of unit tests. So the conversation was done using these regression tests to make sure views were being rendered in the same way.

So each change you see in these static pages is the result of some change you've made. It's all pretty complex, but you should track down where this change is coming from. It could be pointing to unintended side effects of the controller changes.